.all-contributorsrc

```diff
@@ -739,6 +739,15 @@
       "contributions": [
         "code"
       ]
+    },
```

Comment:
You can remove this, as it will be autopopulated by allContributors bot that updates the README as well.